example/ReflectionExample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
```

Comment:
On my machine, BugTrackerExample creates a DLL, not an EXE, so I can't actually run it from the command line.  Something with the target framework being .NETCoreApp?  In any case, I don't need this so I've removed it.